### PR TITLE
Format markdown

### PR DESCRIPTION
**Pull request by mattmoor-sockpuppet**
Produced via: `prettier --write $(find -name '*.md' | grep -v vendor)`

### CODE REVIEW

File: .github/issue-template.md [hunk 1]
```diff
@@ -22,14 +22,10 @@ Classify what kind of issue this is:
...
-
 ## Actual Behavior
 
-
 ## Steps to Reproduce the Problem
 
-  1.
-  2.
-  3.
+1. 2. 3.
```
Comment (vdemeester) [inline .github/issue-template.md]: This doesn't seems right :sweat:

File: .github/pull-request-template.md [hunk 2]
```diff
@@ -2,15 +2,16 @@ Fixes #
 
 ## Proposed Changes
 
-  *
-  *
-  *
+\* \* \*
```
Comment (vdemeester) [inline .github/pull-request-template.md]: This too

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -21,4 +21,4 @@ All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
-[GitHub Help]: https://help.github.com/articles/about-pull-requests/
+[github help]: https://help.github.com/articles/about-pull-requests/
```
Comment (vdemeester) [inline CONTRIBUTING.md]: this too :sweat_smile:

File: .github/issue-template.md (hunk 1 above)
Comment (bobcatfish) [reply .github/issue-template.md]: oh interesting!

File: CONTRIBUTING.md (hunk 3 above)
Comment (bobcatfish) [reply CONTRIBUTING.md]: THE SOCKPUPPET HAS BECOME SENTIENT

**Pull request merged**